egs/wsj/s5/steps/cleanup/create_segments_from_ctm.pl

```diff
@@ -325,7 +325,7 @@ sub ProcessWav {
   # Save the aligned CTM if needed
   if(defined($ACT)){
     for (my $i=0; $i<=$#aligned_ctm; $i++) {
```

Comment:
It currently prints "<word> <start> <duration> <error>", where error is 1 if the word was right in the decoded ctm and 0 otherwise (or the opposite, can't remember).   
I guess it would be nicer to have this a correctly formatted CTM. I just needed the wav id, so I didn't bother making it right.